src/main/java/org/jabref/logic/formatter/Formatters.java

```diff
@@ -42,7 +43,7 @@
     public static final List<Formatter> CASE_CHANGERS = Arrays.asList(
             new CapitalizeFormatter(),
             new LowerCaseFormatter(),
-            new ProtectTermsFormatter(),
+            new ProtectTermsFormatter(Globals.protectedTermsLoader),
```

Comment:
Can you try to remove the dependency on `Globals` from the logic classes? I think it is better to have static constructor methods here (accepting e.g. the terms loader as parameter) instead of static lists. Not sure how many classes are using these fields right now...

Reply:
Turns out this is way harder than expected. There are several other static classes (e.g. `Cleanups`) that depend on this. I'll try to get this extracted somehow, but the current version definitely is not functional.